baseline/OSPS-LE.yaml

```diff
@@ -35,9 +36,9 @@ controls:
     assessment-requirements:
       - id: OSPS-LE-01.01
         text: |
-          The version control system MUST require all code contributors to assert
-          that they are legally authorized to commit the associated contributions
-          on every commit.
+          While active, the version control system MUST require all code
```

Comment:
Does "while active" make sense here?

Reply:
Couldn't think of another "while" statement... got any suggestions?

Reply:
Also I couldn't convince myself to break pattern after doing this like 20 times before :D

Reply:
I'm not sure it needs a while in this case. It's basically `while(True)`. I guess if we want to be sticklers for the format, it's fine, but like...if the project is inactive, it doesn't matter 🤷 

F it. Let's leave it as is and if it causes confusion we can fix it later.